File: templates/languages/csharp/src/formula/formula.cs
using System;
using main;

namespace formula
{
    public class Hello
    {
        public string Input1;
        public string Input2;
        public string Input3;
        public Hello(string sptx, string sptl, string sptb)
        {
            this.Input1 = sptx;
            this.Input2 = sptl;
            this.Input3 = sptb;
            Console.WriteLine("Hello World!");
            Console.WriteLine("You receive " + Input1 + " in text. ");
            Console.WriteLine("You receive " + Input2 + " in list. ");
            Console.WriteLine("You receive " + Input3 + " in boolean. ");
        }

    }
}

File: templates/languages/node/src/formula/formula.js
const clc = require("cli-color")


function Run(input1, input2, input3) {
    console.log("Hello World!")
    console.log(clc.green("You receive "+ input1 +" in text."));
    console.log(clc.red("You receive "+ input2 +" in list."));
    console.log(clc.yellow("You receive "+ input3 +" in boolean."));
}

const formula = Run
module.exports = formula
